definition/primitives.js

```diff
@@ -63,7 +63,7 @@ module.exports = {
     // properties: {
     box: {
       type: 'string',
-      pattern: '\.box$' // was giving me grief so I brutally trimmed it
+      pattern: '.box$' // was giving me grief so I brutally trimmed it
```

Comment:
This was removed with `standard --fix` since periods aren't special characters in strings.

```javascript
'.box$' === '\.box$'
```

Reply:
Don't do this @christianbundy , that string is converted to a regex!
so your "box" string will now accept "somebox" which it should only accept "some.box"

Reply:
Hmm, maybe I should write a test for it? I'd be very surprised if that backslash had any effect, but I've been wrong many times before! :upside_down_face:

Reply:
you are wrong <3

Reply:
and yes, tests are a good idea. I know this because I wrote tests downstream in some dark crystal schema.
our version schema are too loose, so `1N0S0` would pass the pattern `^1.0.0$`. That's not semver!

Reply:
```javascript
RegExp('.box').test('abox')  // => true
RegExp('\.box').test('abox') // => true (these are the same)

RegExp('\\.box').test('abox') // => false (this is what you want)
```

Reply:
Alternatively:

```javascript

RegExp('^1.0.0$').test('1N0S0')  // => true
RegExp('^1\.0\.0$').test('1N0S0') // => true (these are the same)

RegExp('^1\\.0\\.0$').test('1N0S0') // => false (this is what you want)

Reply:
(Fixed the regular expression and added tests in 37b62e0.)

Reply:
nice breakdown, seems like I was wrong?
The thing which isn't clear (because I've not walked the stack) is whether that string does get turned into a regex in the way you proposed. Seems likely it's the case though